vaadin-testbench-integration-tests/pom.xml

```diff
@@ -17,7 +17,6 @@
     <url>http://maven.apache.org</url>
     <properties>
         <jetty.version>9.4.43.v20210629</jetty.version>
```

Comment:
TB 8.2 is used in flow 23.3
jetty version should be 10+.. as in flow 23.3..

Reply:
done